zap/src/main/java/org/zaproxy/zap/extension/api/ContextAPI.java

```diff
@@ -465,6 +466,18 @@ private ApiResponse buildResponseFromContext(Context c) {
         fields.put("inScope", Boolean.toString(c.isInScope()));
         fields.put("excludeRegexs", jsonEncodeList(c.getExcludeFromContextRegexs()));
         fields.put("includeRegexs", jsonEncodeList(c.getIncludeInContextRegexs()));
+        fields.put(
+                "includedTechnologyList",
+                jsonEncodeList(
+                        c.getTechSet().getIncludeTech().stream()
+                                .map(Tech::getUiName)
```

Comment:
Should use `toString` (like the other views).

Reply:
I have changed toString in the latest commit